src/screenipy.py

```diff
@@ -141,102 +202,58 @@ def main(testing=False):
-                    if (executeOption == 1 or executeOption == 3) and (consolidationValue <= ConfigManager.consolidationPercentage and consolidationValue != 0) and isLtpValid:
-                        screenResults = screenResults.append(screeningDictionary,ignore_index=True)
-                        saveResults = saveResults.append(saveDictionary, ignore_index=True)
-                    if executeOption == 4 and isLtpValid and isLowestVolume:
-                        screenResults = screenResults.append(screeningDictionary,ignore_index=True)
-                        saveResults = saveResults.append(saveDictionary, ignore_index=True)
-                    if executeOption == 5 and isLtpValid and isValidRsi:
-                        screenResults = screenResults.append(screeningDictionary,ignore_index=True)
-                        saveResults = saveResults.append(saveDictionary, ignore_index=True)
-                    if executeOption == 6 and isLtpValid:
-                        if reversalOption == 1:
-                            if saveDictionary['Pattern'] in CandlePatterns.reversalPatternsBullish or isMaReversal > 0:
-                                screenResults = screenResults.append(screeningDictionary,ignore_index=True)
-                                saveResults = saveResults.append(saveDictionary, ignore_index=True)
-                        elif reversalOption == 2:
-                            if saveDictionary['Pattern'] in CandlePatterns.reversalPatternsBearish or isMaReversal < 0:
-                                screenResults = screenResults.append(screeningDictionary,ignore_index=True)
-                                saveResults = saveResults.append(saveDictionary, ignore_index=True)
-                    if executeOption == 7 and isLtpValid and isInsideBar:
-                        screenResults = screenResults.append(screeningDictionary,ignore_index=True)
-                        saveResults = saveResults.append(saveDictionary, ignore_index=True)
-                if testing and len(screenResults):
-                    break
-            except KeyboardInterrupt:
-                print(colorText.BOLD + colorText.FAIL + "\n[+] Script terminated by the user." + colorText.END)
-                break
-            except Fetcher.StockDataEmptyException:
-                pass
-            except Exception as e:
-                # print(colorText.FAIL + ("\n[+] Exception Occured while Screening %s! Skipping this stock.." % stock) + colorText.END)
-                if testing:
-                    raise e
+        print(colorText.BOLD + colorText.WARN +
+              "[+] Starting Stock Screening.. Press Ctrl+C to stop!\n")
+
+        items = [(executeOption, reversalOption, daysForLowestVolume, minRSI, maxRSI, respBullBear, insideBarToLookback, stock)
+                 for stock in Fetcher.listStockCodes]
+        results = pool.starmap(screenStocks, items)
```

Comment:
```suggestion
        pool = multiprocessing.Pool(processes=multiprocessing.cpu_count())
        results = pool.starmap(screenStocks, items)
```
`pool` declaration should be moved here because while testing with `pytest` it couldn't find `pool` and throws `NameError`

Reply:
Ok, I will do that. Thank you.